rest/config.go

```diff
@@ -809,6 +809,13 @@ func (config *StartupConfig) Serve(addr string, handler http.Handler) error {
 	)
 }
 
+func (sc *StartupConfig) validateServerProvided() (err error) {
```

Comment:
Worth getting opinion from Mr Config @bbrks but if we're going to any other validation / checking on startup config then it may be worth using the same function.

Reply:
Yes, would be best as my other PR uses it's own validation function as well.

Reply:
Ah okay, if you're doing that in another PR then its probably fine not to need brrks input. Go for it :)